umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/distributed_format_util.clj

```diff
@@ -0,0 +1,134 @@
+    (if-not (boolean (re-find #"\.r.?.? and \.q" formats))
+      (string/split formats #" *and *")
+      formats)
+    formats))
+
+(defn parse-distribution-formats-split-by-and
+  "Split the incomming string or vector of strings by ' and ' except for .r followed by any 2 characters followed by ' and ' and .q plus anything after. The result is a vector of split string if ' and ' exists or the input string if ' and ' does not exist."
+  [formats]
+  (if (instance? String formats)
+    (parse-distribution-formats-split-by-and-input-string formats)
+    (flatten
+      (map parse-distribution-formats-split-by-and-input-string formats))))
+
+(defn parse-distribution-formats-split-by-or-input-string
+  "Split the incomming string by ' or ' The result is a vector of split string if ' or ' exists or the input string if ' or ' does not exist."
+  [formats]
+  (if (string/includes? formats " or ")
+    (string/split formats #" *or *")
+    formats))
+
+(defn parse-distribution-formats-split-by-or
+  "Split the incomming string or vector of strings by ' or '."
+  [formats]
+  (if (instance? String formats)
+    (parse-distribution-formats-split-by-or-input-string formats)
+    (flatten
+      (map parse-distribution-formats-split-by-or-input-string formats))))
+
+(defn parse-distribution-formats-split-by-comma-input-string
+  "Split the incomming string by ', ' The result is a vector of split string if ', ' exists or the input string if ', ' does not exist."
+  [formats]
+  (if (string/includes? formats ",")
+    (string/split formats #", *")
+    formats))
+
+(defn parse-distribution-formats-split-by-comma
+  "Split the incomming string or vector of strings by ', '."
+  [formats]
+  (if (instance? String formats)
+    (parse-distribution-formats-split-by-comma-input-string formats)
+    (flatten
+      (map parse-distribution-formats-split-by-comma-input-string formats))))
+
+(defn parse-distribution-formats-split-by-dash-input-string
+  "Split the incomming string by ' - ' The result is a vector of split string if ' - ' exists or the input string if ' - ' does not exist."
+  [formats]
+  (if (string/includes? formats " - ")
+    (string/split formats #" - ")
+    formats))
+
+(defn parse-distribution-formats-split-by-dash
+  "Split the incomming string or vector of strings by ' - '."
+  [formats]
+  (if (instance? String formats)
+    (parse-distribution-formats-split-by-dash-input-string formats)
+    (flatten
+      (map parse-distribution-formats-split-by-dash-input-string formats))))
+
+(defn parse-distribution-formats-split-by-semicolon-input-string
+  "Split the incomming string by ';' except for ;amp and ;gt. The result is a vector of split string if ';' exists or the input string if ';' does not exist."
+  [formats]
+  (if (string/includes? formats ";")
+    (if-not (or (string/includes? formats ";amp")
+                (string/includes? formats ";gt"))
+      (string/split formats #" *; *")
+      formats)
+    formats))
```

Comment:
This could just be
```(string/split formats #"[;](?!amp)(?!gt)")```

Reply:
I have made the change with a little alteration. " *; *(?!amp)(?!gt)" to also accommodate 0-many spaces before and after the semicolon.